terraso_backend/apps/collaboration/graphql/memberships.py

```diff
@@ -0,0 +1,88 @@
+# Copyright © 2023 Technology Matters
```

Comment:
Yes, that is the idea I don't like the strucutre of moving around by type, it seems better to me if each app handles its own needed files related to graph QL inside their apps, also for this one it seemed more relevant since it can give us some clear limit to where the new collaboration code is without mixing it with the current code, and I think we will leave the graphql app (maybe we can change that name) to handle the schema definition, I will create that issue